### ISSUE TITLE
Using a variable from a global

### ISSUE THREAD

**Issue by wesort** [doublethreedigital/cookie-notice#27]
I'd like set some global variables for the content of the notice and then use those variables within `notice.blade.php`.  `cookies_heading`, `cookies_message`, etc.

## How to reproduce
- Install as per docs including `php artisan vendor:publish --tag=cookie-notice-views`
- Create a `cookies_heading` variable (text field) 
- Try to use that variable with `{{ $settings:cookies_heading }}`
- Result: an exception with "Undefined variable" 

## What I expected
For variable to be available. Everything else is working beautifully.

In Antlers I'd use the variable with `{{ settings:cookies_heading }}`. However, from the [{s} Blade docs](https://statamic.dev/template-engines) I thought `{{ $settings:cookies_heading }}` or even `{{ $title }}` might work but these result in an exception: "Undefined variable" 

## Environment
**Statamic** 3.0.42 Pro
**Laravel** 6.20.16
**PHP** 7.4.9
**doublethreedigital/cookie-notice** 3.0.5
**statamic/ssg** dev-master

**Comment by duncanmcclean** [comment]
Hey, thanks for creating an issue!

So you'd like to be able to use stuff from your globals in your cookie notice banner? `$settings` won't work as we don't pass it in when we render the `notice` view.

**Comment by wesort** [comment]
I don't need the text to necessarily come from globals, but I'd like to give the client a space to edit the text that's shown rather than hardcoding it.

**Comment by duncanmcclean** [comment]
Ah okay, that makes sense!

Currently, there's no way of passing in whatever you want to that view. However, if you want to use a global setting, you should get away with this:

```blade
{{ \Statamic\Facades\GlobalSet::find('name-of-set')->in('default')->get('cookie-notice-text') }}
```

_(in the above example, `name-of-set` is the name of your global set, `default` is the name of your site (as defined in the `sites.php` config file) and `cookie-notice-text` is the handle of the field in your set's blueprint)_

**Comment by wesort** [comment]
That's worked.  Thanks.

As as feature request, it'd be nice to have full access to all variables through standard templating.

**Comment by duncanmcclean** [comment]
Awesome! I'll keep this open as a feature request.

**Comment by duncanmcclean** [comment]
Sorry for the delay! 

I've just moved the templating for the cookie notice view over to Antlers (to feel more familiar to developers who don't work with Laravel). As part of that, you can now use globals inside the view. Here's [a full list of the variables](https://github.com/doublethreedigital/cookie-notice#overriding-the-cookie-notice) which are available.

Released in [v4.0.0](https://github.com/doublethreedigital/cookie-notice/releases/tag/v4.0.0). Have a good weekend!
